### ISSUE TITLE
DVS Gesture learning results are lower than paper's one

### ISSUE THREAD

**Issue by tasbolat1** [bamsumit/slayerPytorch#20]
Hello @bamsumit ,
I have followed the instructions you have given and trained the model by your [architecture.](https://github.com/bamsumit/slayerPytorch/blob/master/exampleLoihi/03_IBMGesture/ibmGestureTrain.py) First, I binned the data with 1450 bins with OR. Second, for training set I chose first 23 subjects, the rest are testing. Here are my network yaml

```
params = {
    "neuron": {
        "type": "SRMALPHA",
        "theta": 10,
        "tauSr": 10.0,
        "tauRef": 1.0,
        "scaleRef": 2,
        "tauRho": 1, # pdf
        "scaleRho": 1,
    },
    "simulation": {"Ts": 1.0, "tSample": 1450, "nSample": 1},
    "training": {
        "error": {
            "type": "NumSpikes",  # "NumSpikes" or "ProbSpikes"
            "tgtSpikeRegion": {  # valid for NumSpikes and ProbSpikes
                "start": 0,
                "stop": 1450,
            },
            "tgtSpikeCount": {True: 180, False: 30},
        }
    },
}
```
However I am unable to get same result as [yours.](https://github.com/bamsumit/slayerPytorch/blob/master/exampleLoihi/03_IBMGesture/TrainedFull/accuracy.png). Here is my result:
![DVS_results_single](https://user-images.githubusercontent.com/4617666/81374092-fdc18f80-9130-11ea-9a62-9438fa67b884.png)

As you can see, the testing accuracy is far below than yours. Is there anything that I am missing?

Kind Regards,
Tasbolat

**Comment by tasbolat1** [comment]
@bamsumit ,
In addition, why there is no psp function? In other examples, there are psps between fc and spike functions.

**Comment by bamsumit** [comment]
Hi, Your second comment has the answer. You should use the PSP function. Otherwise, it is not an SNN, it a binary neural network. The psp can be before dense/conv layer or after. It does not matter as both are linear operations.

My guess is that you are referring to the Loihi example (as you have put links to it). For compatibility with how things work in Loihi, the `psp` and `spike` function are merged in `spikeLoihi`. Please refer to the note on the documentation for [`spikeLoihi`](https://bamsumit.github.io/slayerPytorch/build/html/slayerLoihi.html)

Also, `tauSr = 5` was used in the paper. (I think it is not as detrimental)

Also for this kind of result specific issue, I would appreciate if an issue is not raised. You can email me.
